llama-index-integrations/llms/llama-index-llms-bedrock/llama_index/llms/bedrock/utils.py

```diff
@@ -44,6 +44,7 @@
     "anthropic.claude-v1": 100000,
     "anthropic.claude-v2": 100000,
     "anthropic.claude-v2:1": 200000,
+    "anthropic.claude-3-sonnet-20240229-v1:0": 200000,
```

Comment:
Can we also have `anthropic.claude-3-haiku-20240307-v1:0` here please? It was added in Bedrock a couple days ago..

Reply:
+1